src/components/NcDashboardWidgetItem/NcDashboardWidgetItem.vue

```diff
@@ -197,7 +197,7 @@ export default {
 	}
 	.item__details {
 		padding-left: 8px;
-		max-height: var(--default-clickable-area);
+		max-height: fit-content;
```

Comment:
Yes this is a problem with the new default clickable area, it does not fit 2 lines (well it can but that is quite hacky see the NcSelect for users).
We should still define a maximum, maybe `clickable-area-large`?

Or unify with NcListItem?

Reply:
Yeah, standardizing in the list item sounds good, but won't have time to dive too deep into that. This was just a drive by pr while chatting with Andy hoping for a quick fix 🙈